### PR TITLE
Use new Mother and child measurement slices to generate charts

### PR DESCRIPTION
**Pull request by peterMuriuki**
moves the chart JSX to  new components that pull data from the new chart slices.

### CODE REVIEW

File: src/components/PatientDetailsCharts/ChildChart/index.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    useEffect(() => {
+        const patientId = props.match.params.patient_id;
+        chartSupersetCall(CHILD_CHART_SLICE, addChildDataCreator, supersetService, chartFormDataFilters(patientId))
+            .then(() => {
+                setLoading(false);
+            })
+            .catch((e) => {
+                handleBrokenPage(e);
+            })
+            .finally(() => setLoading(false));
```
Comment (moshthepitt) [inline src/components/PatientDetailsCharts/ChildChart/index.tsx]: Do we need to have `setLoading(false)` twice?

File: src/components/PatientDetailsCharts/MotherChart/index.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    const { error, handleBrokenPage, broken } = useHandleBrokenPage();
+    const [loading, setLoading] = React.useState<boolean>(motherChartData.length === 0);
+    const { t } = useTranslation();
+
+    useEffect(() => {
+        const patientId = props.match.params.patient_id;
+        chartSupersetCall(MOTHER_CHART_SLICE, addMotherDataCreator, supersetService, chartFormDataFilters(patientId))
+            .then(() => setLoading(false))
+            .catch((e) => handleBrokenPage(e))
+            .finally(() => setLoading(false));
```
Comment (moshthepitt) [inline src/components/PatientDetailsCharts/MotherChart/index.tsx]: Doe we need `setLoading(false)` twice?

File: src/components/PatientDetailsCharts/ChildChart/index.tsx (hunk 1 above)
Comment (machariamuguku) [reply src/components/PatientDetailsCharts/ChildChart/index.tsx]: updated 165b8cd730532e6682a47628859a0217374306ae

File: src/components/PatientDetailsCharts/MotherChart/index.tsx (hunk 2 above)
Comment (machariamuguku) [reply src/components/PatientDetailsCharts/MotherChart/index.tsx]: updated 165b8cd730532e6682a47628859a0217374306ae

**Review by machariamuguku — DISMISSED** [review]

**Review by KipSigei — DISMISSED** [review]

**Review by KipSigei — APPROVED** [review]

**Review by moshthepitt — APPROVED** [review]

**Pull request merged**